aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -521,10 +521,11 @@ module aptos_framework::stake {
             coin::merge<AptosCoin>(&mut stake_pool.active, coins);
         };
 
-        let (_minimum_stake, maximum_stake) = staking_config::get_required_stake(&staking_config::get());
-        let total_stake = coin::value<AptosCoin>(&stake_pool.active) +
+        let (_, maximum_stake) = staking_config::get_required_stake(&staking_config::get());
+        let next_epoch_stake = coin::value<AptosCoin>(&stake_pool.active) +
```

Comment:
the name is more confusing than total_stake though, pending_inactive may become inactive if the lockup expires next epoch